### ISSUE TITLE
Correct docu-html in Modelica.Thermal.HeatTransfer.Components.Convection

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1063]
**Reported by leo&#x2e;gall on 3 Apr 2013 15:55 UTC**
Invalid html code seems to break Dymola's view on info layer of Modelica.Thermal.HeatTransfer.Components.Convection.

Documentation annotation should be changed from

```
Re < 5e5 and 0.6 < Pr < 50
```

to

```
Re &lt; 5e5 and 0.6 &lt; Pr &lt; 50
```

(status: MSL trunk r6197)

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1063

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 19 Apr 2013 08:06 UTC**
Fixed in f5752ef9919268507bc11686f0a8aa257264bceb as suggested in ticket

**Comment by modelica-trac-importer** [comment]
**Changelog modified by otter on 19 Apr 2013 08:06 UTC**
Html error corrected in documentation of  Thermal.HeatTransfer.Components.Convection ("&lt;" instead of "<")
